resource-managers/yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnClusterSuite.scala

```diff
@@ -457,9 +463,39 @@ private object YarnClusterDriver extends Logging with Matchers {
             s"Driver logs contain sensitive info (${SECRET_PASSWORD}): \n${log} "
           )
         }
+
+        val yarnConf = new YarnConfiguration(sc.hadoopConfiguration)
+        val httpAddress = System.getenv(ApplicationConstants.Environment.NM_HOST.name()) +
```

Comment:
Use `getNodeManagerHttpAddress`?

Reply:
Ah yes I missed to apply the change to UT as well. Will address.